doc/builder_pattern/Issue_IssueBuilder.puml

```diff
@@ -0,0 +1,26 @@
+@startuml
```

Comment:
Da sind die Dateien der ersten Abgabe auch mit drin.

Reply:
@uhafner Ich verstehe nicht warum, das ist in der 3. Abgabe jetzt auch so. Muss ich die Änderungen immer auf alle repository's pushen oder reicht das analysis-model repo?